### PR TITLE
CASH-1807 add recommended loans row

### PR DESCRIPTION
**Pull request by emuvente**

### CODE REVIEW

**Comment by emuvente** [comment]
Ran into complications with mocking, so there isn't any test. I also ran into issues with the basket id when trying to use the federation endpoint from my vm, so merging this to dev will allow us to actually test things out

**Comment by go-bears** [comment]
🙌  🎉 ,  Woo!

File: src/pages/Lend/LendByCategoryPage.vue [hunk 1]
```diff
@@ -459,23 +522,45 @@ export default {
...
+						query: loanChannelQuery,
+						variables: {
+							// exclude custom rows + limit for ssr
+							ids: _take(_without(ids, ...customCategoryIds), ssrRowLimiter),
+							imgDefaultSize,
+							imgRetinaSize,
+							// @todo variables for fetching data for custom channels
+						},
+					}),
+					recommendedRow ? client.query({
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue]: fun!

File: src/components/LoansByCategory/CategoryRowHover.vue [hunk 2]
```diff
@@ -307,7 +308,7 @@ export default {
 			handler(channel) {
 				this.name = _get(channel, 'name');
 				this.url = _get(channel, 'url');
-				this.loans = _get(channel, 'loans.values');
+				this.loans = _filter(_get(channel, 'loans.values'));
```
Comment (ryan-ludwig) [inline src/components/LoansByCategory/CategoryRowHover.vue]: what does this _filter do?

Comment (emuvente) [reply src/components/LoansByCategory/CategoryRowHover.vue]: Handles cases where `loans` is something like `[null, null]` (I only encountered this in testing)

**Pull request merged**